lib/definitions/project.d.ts

```diff
@@ -26,13 +26,17 @@ interface IPlatformProjectServiceBase {
 	getPluginPlatformsFolderPath(pluginData: IPluginData, platform: string): string;
 }
 
+interface IBuildConfig {
+	runSbGenerator?: boolean;
```

Comment:
Because it is still experimental option. Once we prove it is stable, we will switch it to be on by default - possibly for {N} 1.4.